src/index.js

```diff
@@ -1,93 +1,98 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
-
-const isProduction = process.env.NODE_ENV === "production";
+const fs = require('fs'),
```

Comment:
> Looks good to me....Hound already flag the suggestions I would have raised. So I think you should fix them.

I've resolved the ones pertaining to linting. As for the unused vars I cannot remove them on my own, since it isn't my place to remove or add modules.

Reply:
I am not sure why you touched these lines, you have these hound issues cos of that. Please revert these linting changes that led to these new hound errors. I would have suggested you go ahead and fix these hound issues but someone else is already handling that task.